src/PrestaShopBundle/Resources/config/routing/admin/sell/customer_service/merchandise_return.yml

```
@@ -23,28 +25,69 @@ admin_merchandise_returns_filter:
     gridDefinitionFactoryServiceId: prestashop.core.grid.definition.factory.merchandise_return
     redirectRoute: admin_merchandise_returns_index
     _legacy_controller: AdminReturn
-#  _legacy_link: AdminReturn:submitFilterorder_return
+    _legacy_link: AdminReturn:submitFilterorder_return
+    _legacy_feature_flag: order_return
 
 admin_order_returns_edit:
   path: /{orderReturnId}/edit
-  methods: [ GET, POST ]
+  methods: [ GET ]
   defaults:
     _controller: PrestaShopBundle\Controller\Admin\Sell\CustomerService\MerchandiseReturnController::editAction
     _legacy_controller: AdminReturn
-    #    _legacy_link: AdminReturn:updateorder_return
+    _legacy_link: AdminReturn:updateorder_return
+    _legacy_feature_flag: order_return
     _legacy_parameters:
       id_order_return: orderReturnId
   requirements:
     orderReturnId: \d+
 
+admin_order_returns_products_filter:
```

not a big deal, but I think we are always referring as `search` and not `filter`? (+ the action is called searchGrid, so in theory it should be `...search_grid:`, but search should be ok as well, as there are lots usages like this :man_shrugging: 
```suggestion
admin_order_return_products_search:
```